### PR TITLE
https://issues.redhat.com/browse/ACM-7820 Backup restore considerations

### PR DESCRIPTION
**Pull request by dockerymick**
https://issues.redhat.com/browse/ACM-7820

### CODE REVIEW

File: business_continuity/backup_restore/backup_intro.adoc [hunk 1]
```diff
@@ -18,25 +18,11 @@ Continue reading the following topics to learn more about the backup and restore
...
 
 * xref:../backup_restore/backup_validate.adoc#backup-validation-using-a-policy[Validating your backup or restore configurations]
-** xref:../backup_restore/backup_validate.adoc#protecting-data-using-server-side-encryption[Protecting data using server-side encryption]
 
 * xref:../backup_restore/backup_msa.adoc#auto-connect-clusters-msa[Connecting clusters automatically by using a Managed Service Account]
-** xref:../backup_restore/backup_msa.adoc#enabling-auto-import[Enabling automatic import]
-** xref:../backup_restore/backup_msa.adoc#limitations-auto-import[Automatic import limitations]
-** xref:../backup_restore/backup_msa.adoc#disabling-auto-import[Disabling automatic import]
 
 * xref:../backup_restore/backup_adv_config.adoc#dr4hub-advanced-configuration[Backup and restore advanced configuration]
```
Comment (dockerymick) [inline business_continuity/backup_restore/backup_intro.adoc:28]: Removed subtopics from the list on the intro file

**Review by birsanv — APPROVED** [review]
lgtm

**Review by oafischer — APPROVED** [review]
Looks good, thanks for cleaning up!

File: business_continuity/backup_restore/backup_msa.adoc [hunk 2]
```diff
@@ -70,8 +70,8 @@ After you enable the automatic import feature, the backup component starts proce
 
 The token is only created if the managed cluster is accessible when you create the Managed Service Account, otherwise it is created later once the managed cluster becomes available.
 
-[#limitations-auto-import]
-== Automatic import limitations
+[#auto-import-considerations]
+== Automatic import considerations
 
 The following situations can prevent the managed cluster from being automatically imported when moving to a new hub cluster:
```
Comment (dockerymick) [inline business_continuity/backup_restore/backup_msa.adoc]:
```suggestion
The following scenarios can prevent the managed cluster from being automatically imported when moving to a new hub cluster:
```

**Pull request merged**